lib/TBDGen/TBDGen.cpp

```diff
@@ -1021,6 +1021,10 @@ void TBDGenVisitor::visitEnumDecl(EnumDecl *ED) {
   visitNominalTypeDecl(ED);
 }
 
+void TBDGenVisitor::visitStructDecl(StructDecl *SD) {
+  visitNominalTypeDecl(SD);
+}
```

Comment:
Is this change covered by an existing failing test?

Reply:
I'm not really sure where this came from. I removed the change and all the tests still pass.